### DISCUSSION TITLE
Z seams on A1 mini during vase/spiral mode

### DISCUSSION THREAD

**Discussion by Brandon5799** [OrcaSlicer/OrcaSlicer#7725] [Q&A]
So I ended up getting an A1 mini while they are on sale. Went to print a small vase print and had horrible z seams. I thought i forgot to select smooth spiral but it was selected. So as a test I sliced in Bambu Studios and no seam. Am I missing something simple here, it’s like the old marlin days I can see a slight pause at each start of each layer for the power recovery glitch that left blobs. On my p1S no seams in Bambu or orca as long as I have the smooth spiral selected but it’s not working on my mini but just in orca. Is this  known bug or me lol?

**Comment by pgifford** [comment]
Your A1 Mini 3MF file would be helpful, can you attach it?  If I get time I'll try it on my A1 Mini to see what I get.
